.github/workflows/build.yml

```diff
@@ -1,10 +1,34 @@
-name: Build Zed Nightly
+name: Build
 
 on:
-  schedule:
-    - cron: "0 0 * * *" # Runs every night at midnight UTC
+  workflow_call:
+    inputs:
+      repository:
+        description: Repository to clone and build
+        type: string
+        default: zed-industries/zed
+      ref:
+        description: Branch/tag/ref to build
+        type: string
+        required: true
+      patch:
+        description: Apply someone120's (Yakira's) patch fixing LSP server discovery
+        type: boolean
+        default: true
   workflow_dispatch:
-  pull_request:
+    inputs:
+      repository:
+        description: Repository to clone and build
+        type: string
+        default: zed-industries/zed
+      ref:
+        description: Branch/tag/ref to build
+        type: string
+        default: main
```

Comment:
The build workflow can still be run from the UI, now with the ability to specify which repository (fork of zed-industries/zed) and ref to use. This is helpful for the community to try out fixes and personalizations on windows. Rest assured, no releases are created by this workflow anymore, so running from UI will only result in artifacts only.

Reply:
This is a great addition. 👌